Formula/glibc.rb

```diff
@@ -94,22 +56,59 @@ class Glibc < Formula
   end
 
   depends_on "binutils" => :build
-  depends_on GawkRequirement => :build
-  depends_on "linux-headers@4.4" => :build
-  depends_on MakeRequirement => :build
-  depends_on SedRequirement => :build
   depends_on BrewedGlibcNotOlderRequirement
   depends_on :linux
+  depends_on "linux-headers@5.15"
   depends_on LinuxKernelRequirement
 
-  # GCC 4.7 or later is required.
-  fails_with gcc: "4.6"
+  resource "bootstrap-bison" do
+    url "https://github.com/danielnachun/homebrew-bootstrap/releases/download/v1.0.0/bootstrap-bison-3.8.2.tar.gz"
+    sha256 "f54af1fb828434e98251e3bfad30d96934858c9b04866382379a17b99f88c605"
+  end
+
+  resource "bootstrap-gawk" do
+    url "https://github.com/danielnachun/homebrew-bootstrap/releases/download/v1.0.0/bootstrap-gawk-5.1.1.tar.gz"
+    sha256 "0a5850696c0fc253eaeb233b9720b9610092f87de69c1caf610fd5fbf4aa8a3b"
+  end
+
+  resource "bootstrap-gcc" do
+    url "https://github.com/danielnachun/homebrew-bootstrap/releases/download/v1.0.0/bootstrap-gcc-9.5.0.tar.gz"
+    sha256 "234e65deabdec5d7a208848d6f77d42e9f1e62638cdf39dd11c557278006e315"
+  end
+
+  resource "bootstrap-make" do
+    url "https://github.com/danielnachun/homebrew-bootstrap/releases/download/v1.0.0/bootstrap-make-4.3.tar.gz"
+    sha256 "fea3a153aef4afbdfe3702beeda8c3f1583e192f20daeea0cb744d8c9af11771"
+  end
+
+  resource "bootstrap-python3" do
+    url "https://github.com/danielnachun/homebrew-bootstrap/releases/download/v1.0.0/bootstrap-python3-3.9.13.tar.gz"
+    sha256 "c2d4f708cff89a837de8efaa9c46d601ba50ce6aa95681bfb5630019decdc6fe"
+  end
+
+  resource "bootstrap-sed" do
+    url "https://github.com/danielnachun/homebrew-bootstrap/releases/download/v1.0.0/bootstrap-sed-4.8.tar.gz"
+    sha256 "e5b94524bc868ac7e4855a2a3ee992ea0b5f9e5c0711e89490bd0fa3d9d607f8"
+  end
 
   def install
-    # Fix Error: `loc1@GLIBC_2.2.5' can't be versioned to common symbol 'loc1'
-    # See https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=869717
-    # Fixed in glibc 2.24
-    inreplace "misc/regexp.c", /^(char \*loc[12s]);$/, "\\1 __attribute__ ((nocommon));"
+    # Set up bootstrap resources in /tmp/homebrew.
+    bootstrap_dir = Pathname.new("/tmp/homebrew")
+    mkdir bootstrap_dir
+
+    resources.each do |r|
+      r.stage do
+        cp_r Pathname.pwd.children, bootstrap_dir
+      end
```

Comment:
```suggestion
      r.stage bootstrap_dir
```
I take it this doesn't work?

Reply:
Yeah I hit a really weird issue when I tried it that way.  The first resource installs fine, but then the subsequent resource tries to create its directories **inside** of the directories of the first one, so if `/tmp/homebrew/lib` exists, the second resource makes `/tmp/homebrew/lib/lib`.  Then the third resource fails because it tries to create `/tmp/homebrew/lib/lib` but that already exists.  

I'm not sure if this behavior is intended or is a bug, but the only way to make it work that I've found so far is to stage the resource in it's own directory then copy it to `/tmp/homebrew`.  However I feel like `r.stage bootstrap_dir` should "just work", unless I'm misunderstanding its intended usage.

Reply:
Interesting. What about
```ruby
bootstrap_dir.install r
```
?

Reply:
Sadly it does not work either.  If we can eventually find a cleaner way to do this I'm totally in favor of doing so, but we shouldn't block merging this on that issue.